packages/roosterjs-editor-core/lib/editor/Editor.ts

```diff
@@ -483,6 +498,27 @@ export default class Editor implements IEditor {
         return !!range && this.contains(range) && core.api.selectRange(core, range);
     }
 
+    private getImageSelection(
+        core: EditorCore,
+        arg: Range | NodePosition | Node | SelectionPath | HTMLTableElement | null
+    ) {
+        if (safeInstanceOf(arg, 'HTMLImageElement')) {
+            const selection = core.api.selectImage(core, arg);
+            return selection;
+        }
+        if (arg && safeInstanceOf(arg, 'HTMLSpanElement')) {

```

Comment:
This is specially for image edit plugin, right?

I feel we can change the way how image edit plugin show the border, then we can get rid of this part.

And once we remove this part, no need to have this private function, but just put everything inside select function